cluster/gce/util.sh

```diff
@@ -225,6 +232,36 @@ function upload-server-tars() {
   SALT_TAR_URL="${salt_gs_url/gs:\/\//https://storage.googleapis.com/}"
 }
 
+# Figure out which binary use on the server and assure it is available.
+# If KUBE_VERSION is specified use binaries specified by it, otherwise
+# use local dev binaries.
+#
+# Assumed vars:
+#   PROJECT
+# Vars set:
+#   SERVER_BINARY_TAR_URL
+#   SALT_TAR_URL
+function tars_from_version() {
```

Comment:
I would still lift this function out and put it in [cluster/common.sh](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cluster/common.sh). I would also lift the `set_binary_version` function out of `upgrade.sh` out, because it gives you the symbolic names. As I mentioned here: https://github.com/GoogleCloudPlatform/kubernetes/pull/8789#discussion_r31234663, these are k8s Release Engineering files, and generally applicable across providers.